api.swagger.yaml

```diff
@@ -1845,6 +1845,51 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/manuals/{manual_page_id}/pages':
+    get:
+      description: Get all pages in a manual or manual page.
+      tags:
+        - manuals
+      parameters:
```

Comment:
path parameter `manual_page_id` is missing

Reply:
Thanks; fixed.